### ISSUE TITLE
Cannot download sample of dataset

### ISSUE THREAD

**Issue by dogTK** [broadinstitute/CellBender#214]
### issue
When launching generate_tiny_10x_pbmc.py, which is located under CellBender/examples/remove_background/, an error occurs due to the dataset not being downloaded.

```
(base) hoge@dogTK remove_background % pwd
/Users/hoge/Desktop/CellBender/examples/remove_background
(base) hoge@dogTK remove_background % python generate_tiny_10x_pbmc.py
Downloading pbmc4k (CellRanger 2.1.0, v2 Chemistry)...
Could not retrieve pbmc4k (CellRanger 2.1.0, v2 Chemistry) -- cannot continue!
(base) hoge@dogTK remove_background % 
```

**Comment by sjfleming** [comment]
Yes, you're right.  I have made an attempt to address that here:
https://github.com/broadinstitute/CellBender/blob/5936fc9301691569761bcc51f41f14d7f79f34d5/examples/remove_background/generate_tiny_10x_dataset.py#L25

and this will go into the v0.3.0 release.

**Comment by sjfleming** [comment]
Your solution in #216 might be better

**Comment by dogTK** [comment]
> this will go into the v0.3.0 release.

That sounds good!
I'm looking forward to using it. Thanks!

**Comment by sjfleming** [comment]
Closed by #238
